homeassistant/components/opower/sensor.py

```diff
@@ -71,6 +72,8 @@ class OpowerEntityDescription(SensorEntityDescription, OpowerEntityDescriptionMi
         key="elec_cost_to_date",
         name="Current bill electric cost to date",
         device_class=SensorDeviceClass.MONETARY,
+        native_unit_of_measurement=CURRENCY_DOLLAR,
```

Comment:
The monetary device class sensors should use the ISO three letter character code as unit of measurement.

```suggestion
        native_unit_of_measurement="USD",
```